packages/bot/src/common/behaviors/dice/dice.behavior.base.ts

```diff
@@ -146,6 +153,40 @@ export abstract class DiceBehaviorBase extends BehaviorBase {
     return null;
   }
 
+
+
+  protected rollSpoileredV1(trigger: Trigger, logTag: string, content: string, context: BehaviorContext, requireDice: boolean): string[] | null {
+    const match = content.match(/([sS]#)(?:(\d+)#\s*)?(.*)/i);
+
+    if (!match) {
+      return null;
+    }
+    const diceWithoutSpoiler = match[3] ? match[3] : content;
+
+
+    const lines: string[] = [];
+    const result = this.parsers.v1.tryParse(diceWithoutSpoiler);
+    if (!result) { return null; }
+
+    // don't be too aggressive with the replies
+    const hasEnoughDice = requireDice ? result.dice > 0 : true;
+    const hasPrefix = isKnownPrefix(context.messageConfiguredOptions?.prefixStyle);
+    const shouldReply = hasPrefix || (result.depth > 1 && hasEnoughDice);
+    if (!shouldReply) { return null; }
+
+    const response = this.buildMessage(result, requireDice);
+
+    if (response && shouldReply) {
+      lines.push("||" + response + "||");
+    }
+      this.promLogger.incParserUse(ParserRollType.Spoilered, 'v1');
```

Comment:
Note for me later: indentation here seems weird. Not sure if that's related to the PR viewer, but either way I should probably set up a default code formatter.

Reply:
https://github.com/rollem-discord/rollem-discord/issues/228